README.md

```diff
@@ -87,20 +90,51 @@ Consider the case where a `Span`'s lifetime logically starts in one execution co
 ------------------------------------------------> time
 ```
 
-The `ServiceHandlerSpan` is _active_ when it's running FunctionA and FunctionB, and inactive while it's waiting on an RPC (presumably modelled as its own Span, though that's not the concern here).
+The `"ServiceHandlerSpan"` is _active_ when it's running FunctionA and FunctionB, and inactive while it's waiting on an RPC (presumably modelled as its own Span, though that's not the concern here).
+
+**The `ActiveSpanSource` makes it easy to "adopt" the Span and execution context in `FunctionA` and re-activate it in `FunctionB`.** These are the steps:
+
+1. Start the `Span` via `Tracer.startAndActivate()` rather than via `Tracer.start()`; or, if the `Span` was already `start()`ed, call `ActiveSpanSource#adopt(span)`. Either route will yield an `ActiveSpanSource.Handle` instance that's "adopted" the `Span`.
+2. In the method/function that *allocates* the closure/`Runnable`/`Future`/etc, call `ActiveSpanSource.Handle#defer()` to obtain an `ActiveSpanSource.Continuation`
+3. In the closure/`Runnable`/`Future`/etc itself, invoke `ActiveSpanSource.Continuation#activate` to re-activate the `ActiveSpanSource.Handle`, then `deactivate()` it (or use try-with-resources for less typing).
+
```

Comment:
So many verbs in a small bit of text... _start, activate, adopt, allocate, defer, re-activate_ and _deactivate_ ...
By the way, I think they're all **much** better chosen than previous iterations, but I think a reader trying to discover what OpenTracing is will get absolutely overwhelmed by these couple of bullets :wink: